### PR TITLE
gnome-base/gnome-desktop: Version bump to 43.0

### PR DESCRIPTION
**Pull request by Schievel1**
Signed-off-by: Pascal Jäger <pascal.jaeger@leimstift.de>

### CODE REVIEW

**Comment by mattst88** [comment]
>  gnome-base/gnome-desktop: Version bump to 43.0

Ugh, the version is actually just `43`, not `43.0`. I haven't seen any other GNOME project do this.

File: gnome-base/gnome-desktop/gnome-desktop-43-r300.ebuild [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		-Dlegacy_library=true
+	)
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+
+	rm -r \
+		"${ED}"/usr/share/locale \
```
Comment (mattst88) [inline gnome-base/gnome-desktop/gnome-desktop-43-r300.ebuild:73]: Looks like you dropped `gnome-version.xml` here:

````diff
 	rm -r \
-		"${ED}"/usr/share/gnome/gnome-version.xml \
 		"${ED}"/usr/share/locale \
 		|| die
````

Looks good, I see this in the `NEWS` file: `- Remove gnome-version.xml`. Cool. :)

**Review by mattst88 — APPROVED** [review]
This looks good. I'll give it a test soon and commit it if everything goes well. Nice work!

File: gnome-base/gnome-desktop/gnome-desktop-43-r300.ebuild (hunk 1 above)
Comment (Schievel1) [reply gnome-base/gnome-desktop/gnome-desktop-43-r300.ebuild:73]: Yup, file didn't exist when I tried it s no need to remove it

**Comment by Schievel1** [comment]
> > gnome-base/gnome-desktop: Version bump to 43.0
> 
> Ugh, the version is actually just `43`, not `43.0`. I haven't seen any other GNOME project do this.

Oh yes, that is why I named the ebuild like this. Forgot it in the commit message and PR title, though. 
Is it ok or should I correct the commit message and PR?

**Comment by mattst88** [comment]
Nah, don't worry about it.

File: gnome-base/gnome-desktop/gnome-desktop-43-r400.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+EAPI=8
+inherit gnome.org meson xdg
+
+DESCRIPTION="Library with common API for various GNOME modules"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-desktop/"
+
+LICENSE="GPL-2+ LGPL-2+ FDL-1.1+"
+SLOT="4/1" # subslot = libgnome-desktop-4 soname version
+IUSE="debug gtk-doc seccomp systemd udev introspection"
```
Comment (mattst88) [inline gnome-base/gnome-desktop/gnome-desktop-43-r400.ebuild:12]: `IUSE` should always be alphabetical, but... I intentionally dropped the `introspection` USE flag from the -r400 ebuild in

````
commit baf18a93c451c680657b01e7483027fb853cc090
Author: Matt Turner <mattst88@gentoo.org>
Date:   Mon Jul 11 22:59:18 2022 -0700

    gnome-base/gnome-desktop: Version bump to 42.3
    
    * Drop IUSE=introspection from -r400 (bug #856649)
    
    Closes: https://bugs.gentoo.org/856649
    Signed-off-by: Matt Turner <mattst88@gentoo.org>
````

I don't believe it's possible to disable introspection in -r400, and with `USE=-introspection` the build fails exactly the same way as it did as reported in that bug.

File: gnome-base/gnome-desktop/gnome-desktop-43-r400.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome.org meson xdg
+
+DESCRIPTION="Library with common API for various GNOME modules"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-desktop/"
+
+LICENSE="GPL-2+ LGPL-2+ FDL-1.1+"
+SLOT="4/1" # subslot = libgnome-desktop-4 soname version
```
Comment (mattst88) [inline gnome-base/gnome-desktop/gnome-desktop-43-r400.ebuild:11]: Looks like the soversion changed to `2`.

**Review by mattst88 — CHANGES_REQUESTED** [review]

**Review by mattst88 — APPROVED** [review]

**Pull request closed without merging**